README.md

```diff
@@ -13,7 +13,7 @@ a known GOOD or BAD server.
 In order to run this script you have to allow local PowerShell scripts to run.
 To do this: Open a PowerShell as Administrator and then enter:
 
-    C:> Set-ExecutionPolicy remotesigned
+    C:> Set-ExecutionPolicy remotesigned -scope process
```

Comment:
Hmm, but does this also mean that the script must be run from the same console? In that case it's way less user friendly than now. I'm not against this change, however are there different ways to achieve a safe(r) way? E.g. enable scripts only until the next reboot?